BridgeSDK.xcodeproj/xcshareddata/xcschemes/BridgeSDK.xcscheme

```diff
@@ -58,6 +58,18 @@
             ReferencedContainer = "container:BridgeSDK.xcodeproj">
          </BuildableReference>
       </MacroExpansion>
+      <EnvironmentVariables>
+         <EnvironmentVariable
+            key = "SAGE_ADMIN_EMAIL"
+            value = "bridgeit@sagebase.org"
+            isEnabled = "YES">
+         </EnvironmentVariable>
+         <EnvironmentVariable
+            key = "SAGE_ADMIN_PASSWORD"
+            value = "apB988xEUv&lt;"
```

Comment:
dangit, I reverted that before making the original PR commit. How did it come back? Well, fixed (again)